.github/CONTRIBUTING.md

```diff
@@ -4,72 +4,149 @@ Water.css becomes better for everyone when people like you help make it better!
 
 Have any questions or concerns? Did I forget an element or selector? Does something look ugly? Feel free to submit an issue or pull request.
 
-Before contributing, please read the [code of conduct](CODE_OF_CONDUCT.md). Also you agree that your contributions will be licensed under its [MIT License](./LICENSE.md).
+Before contributing, please read the [code of conduct](CODE_OF_CONDUCT.md). Also, you agree that your contributions will be licensed under this project's [MIT License](../LICENSE.md).
 
-## How to get started
+## Overview
 
-1. Get a copy of repository. It is recommended to fork it first and clone to your machine using `git`.
+Please take a moment to read through the following guidelines:
 
-2. Make sure that you have [yarn](https://yarnpkg.com) and install dependencies listed in `package.json` using it.
+- [Get started](#get-started)
+- [Find issues to work on](#find-issues-to-work-on)
+- [Add or change styles](#add-or-change-styles)
+- [Create your pull request](#create-your-pull-request)
+- [Project structure](#project-structure)
 
-```
-yarn
-```
+<br>
 
-3. Then you can run development server with live reloading out of the box and play around with the framework.
+> **TL;DR**:
+> 1. `yarn install` – install dependencies
+> 2. `yarn dev` – start the dev server, make your changes
+> 3. `yarn validate` – make sure your changes pass all tests
```

Comment:
I ran `yarn validate` on my Windows 10 computer and got:
```
✖ 446 problems (446 errors, 0 warnings)
  409 errors and 0 warnings potentially fixable with the `--fix` option.
```
which was finding errors in both `index.html` and `script.js` 😬 Not sure if I'm doing something wrong

Reply:
Oh, interesting! Those issues might be about line endings then?
I know the default prettier config enforces LF endings, which can be a bit annoying when you're using the Prettier extension and have configured your git to convert to and from CRLF on checkout/commit. Don't know if our rules (standard?) have something similar

Reply:
Here's the full log, I don't think it's due to the line endings.
<details>
<summary>Full Log</summary>

```
PS D:\Development\water.css> yarn validate
yarn run v1.22.4
$ yarn lint:js && yarn lint:css && yarn accessibility
$ eslint **/*.js **/*.html gulpfile.js

D:\Development\water.css\dist\docs\index.html
  1:2161  error  Operator '||' must be spaced                                           space-infix-ops
  1:2274  error  Unnecessary escape character: \/                                       no-useless-escape
  1:2285  error  Extra semicolon                                                        semi
  1:2286  error  Newline required at end of file but not found                          eol-last
  8:126   error  Operator '=' must be spaced                                            space-infix-ops
  8:143   error  Operator '||' must be spaced                                           space-infix-ops
  8:147   error  Missing whitespace after semicolon                                     semi-spacing
  8:158   error  Operator '=' must be spaced                                            space-infix-ops
  8:159   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  8:167   error  Missing space before function parentheses                              space-before-function-paren
  8:169   error  Requires a space after '{'                                             block-spacing
  8:169   error  Missing space before opening brace                                     space-before-blocks
  8:202   error  Requires a space before '}'                                            block-spacing
  8:203   error  Missing whitespace after semicolon                                     semi-spacing
  8:204   error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  8:209   error  Strings must use singlequote                                           quotes
  8:213   error  A space is required after ','                                          comma-spacing
  8:214   error  Missing '()' invoking a constructor                                    new-parens
  8:223   error  Unexpected use of comma operator                                       no-sequences
  8:223   error  A space is required after ','                                          comma-spacing
  8:229   error  Strings must use singlequote                                           quotes
  8:237   error  A space is required after ','                                          comma-spacing
  8:238   error  Strings must use singlequote                                           quotes
  8:255   error  Extra semicolon                                                        semi
  8:256   error  Newline required at end of file but not found                          eol-last

D:\Development\water.css\dist\docs\script.js
  1:1     error  Strings must use singlequote                                           quotes
  1:13    error  Missing whitespace after semicolon                                     semi-spacing
  1:14    error  Expected a function expression                                         func-style
  1:14    error  Use const or class constructors instead of named functions             prefer-arrow/prefer-arrow-functions
  1:24    error  Missing space before function parentheses                              space-before-function-paren
  1:26    error  A space is required after ','                                          comma-spacing
  1:29    error  Requires a space after '{'                                             block-spacing
  1:29    error  Missing space before opening brace                                     space-before-blocks
  1:35    error  Missing whitespace after semicolon                                     semi-spacing
  1:36    error  Expected space(s) after "if"                                           keyword-spacing
  1:39    error  Expected literal to be on the right side of ==                         yoda
  1:39    error  Strings must use singlequote                                           quotes
  1:50    error  Expected '===' and instead saw '=='                                    eqeqeq
  1:50    error  Operator '==' must be spaced                                           space-infix-ops
  1:65    error  Operator '||' must be spaced                                           space-infix-ops
  1:67    error  Expected literal to be on the right side of ==                         yoda
  1:71    error  Operator '==' must be spaced                                           space-infix-ops
  1:92    error  Requires a space after '{'                                             block-spacing
  1:92    error  Missing space before opening brace                                     space-before-blocks
  1:93    error  Expected space(s) after "if"                                           keyword-spacing
  1:112   error  Operator '||' must be spaced                                           space-infix-ops
  1:116   error  Operator '=' must be spaced                                            space-infix-ops
  1:117   error  Wrap an immediate function invocation in parentheses                   wrap-iife
  1:117   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:125   error  Missing space before function parentheses                              space-before-function-paren
  1:127   error  A space is required after ','                                          comma-spacing
  1:130   error  Requires a space after '{'                                             block-spacing
  1:130   error  Missing space before opening brace                                     space-before-blocks
  1:131   error  Expected space(s) after "if"                                           keyword-spacing
  1:137   error  Expected space(s) before "return"                                      keyword-spacing
  1:143   error  Missing whitespace after semicolon                                     semi-spacing
  1:144   error  Expected space(s) after "if"                                           keyword-spacing
  1:147   error  Expected literal to be on the right side of ==                         yoda
  1:147   error  Strings must use singlequote                                           quotes
  1:155   error  Expected '===' and instead saw '=='                                    eqeqeq
  1:155   error  Operator '==' must be spaced                                           space-infix-ops
  1:166   error  Expected space(s) before "return"                                      keyword-spacing
  1:176   error  A space is required after ','                                          comma-spacing
  1:179   error  Missing whitespace after semicolon                                     semi-spacing
  1:185   error  Operator '=' must be spaced                                            space-infix-ops
  1:227   error  A space is required after ','                                          comma-spacing
  1:231   error  Missing whitespace after semicolon                                     semi-spacing
  1:232   error  Expected literal to be on the right side of ===                        yoda
  1:232   error  Strings must use singlequote                                           quotes
  1:240   error  Operator '===' must be spaced                                          space-infix-ops
  1:244   error  Operator '&&' must be spaced                                           space-infix-ops
  1:259   error  Operator '&&' must be spaced                                           space-infix-ops
  1:263   error  Operator '=' must be spaced                                            space-infix-ops
  1:283   error  Missing whitespace after semicolon                                     semi-spacing
  1:284   error  Expected space(s) after "if"                                           keyword-spacing
  1:287   error  Expected literal to be on the right side of ===                        yoda
  1:287   error  Strings must use singlequote                                           quotes
  1:292   error  Operator '===' must be spaced                                          space-infix-ops
  1:296   error  Operator '||' must be spaced                                           space-infix-ops
  1:298   error  Expected literal to be on the right side of ===                        yoda
  1:298   error  Strings must use singlequote                                           quotes
  1:303   error  Operator '===' must be spaced                                          space-infix-ops
  1:308   error  Expected space(s) before "return"                                      keyword-spacing
  1:328   error  Missing whitespace after semicolon                                     semi-spacing
  1:329   error  Expected space(s) after "if"                                           keyword-spacing
  1:332   error  Expected literal to be on the right side of ===                        yoda
  1:332   error  Strings must use singlequote                                           quotes
  1:343   error  Operator '===' must be spaced                                          space-infix-ops
  1:347   error  Operator '||' must be spaced                                           space-infix-ops
  1:400   error  Expected space(s) before "return"                                      keyword-spacing
  1:410   error  A space is required after ','                                          comma-spacing
  1:413   error  Requires a space before '}'                                            block-spacing
  1:418   error  Operator '||' must be spaced                                           space-infix-ops
  1:418   error  Unexpected mix of '||' and '&&'                                        no-mixed-operators
  1:421   error  Operator '&&' must be spaced                                           space-infix-ops
  1:424   error  Unexpected mix of '||' and '&&'                                        no-mixed-operators
  1:424   error  Operator '&&' must be spaced                                           space-infix-ops
  1:426   error  Expected literal to be on the right side of ==                         yoda
  1:426   error  Strings must use singlequote                                           quotes
  1:434   error  Expected '===' and instead saw '=='                                    eqeqeq
  1:434   error  Operator '==' must be spaced                                           space-infix-ops
  1:452   error  Requires a space after '{'                                             block-spacing
  1:452   error  Missing space before opening brace                                     space-before-blocks
  1:454   error  Operator '&&' must be spaced                                           space-infix-ops
  1:458   error  Operator '=' must be spaced                                            space-infix-ops
  1:461   error  Missing whitespace after semicolon                                     semi-spacing
  1:462   error  Split initialized 'var' declarations into multiple statements          one-var
  1:467   error  Operator '=' must be spaced                                            space-infix-ops
  1:469   error  A space is required after ','                                          comma-spacing
  1:471   error  Operator '=' must be spaced                                            space-infix-ops
  1:472   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:480   error  Missing space before function parentheses                              space-before-function-paren
  1:482   error  Missing space before opening brace                                     space-before-blocks
  1:484   error  Missing whitespace after semicolon                                     semi-spacing
  1:485   error  Expected space(s) after "return"                                       keyword-spacing
  1:491   error  A space is required after '{'                                          object-curly-spacing
  1:494   error  Missing space before value for key 's'                                 key-spacing
  1:495   error  A space is required after ','                                          comma-spacing
  1:498   error  Missing space before value for key 'n'                                 key-spacing
  1:498   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:506   error  Missing space before function parentheses                              space-before-function-paren
  1:508   error  Requires a space after '{'                                             block-spacing
  1:508   error  Missing space before opening brace                                     space-before-blocks
  1:517   error  Operator '>=' must be spaced                                           space-infix-ops
  1:527   error  Operator '?' must be spaced                                            space-infix-ops
  1:528   error  A space is required after '{'                                          object-curly-spacing
  1:534   error  Missing space before value for key 'done'                              key-spacing
  1:536   error  A space is required before '}'                                         object-curly-spacing
  1:538   error  A space is required after '{'                                          object-curly-spacing
  1:544   error  Missing space before value for key 'done'                              key-spacing
  1:546   error  A space is required after ','                                          comma-spacing
  1:553   error  Missing space before value for key 'value'                             key-spacing
  1:559   error  A space is required before '}'                                         object-curly-spacing
  1:560   error  Requires a space before '}'                                            block-spacing
  1:561   error  A space is required after ','                                          comma-spacing
  1:564   error  Missing space before value for key 'e'                                 key-spacing
  1:564   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:572   error  Missing space before function parentheses                              space-before-function-paren
  1:575   error  Requires a space after '{'                                             block-spacing
  1:575   error  Missing space before opening brace                                     space-before-blocks
  1:583   error  Requires a space before '}'                                            block-spacing
  1:584   error  A space is required after ','                                          comma-spacing
  1:587   error  Missing space before value for key 'f'                                 key-spacing
  1:588   error  A space is required before '}'                                         object-curly-spacing
  1:589   error  Requires a space before '}'                                            block-spacing
  1:590   error  Expected space(s) before "throw"                                       keyword-spacing
  1:610   error  Strings must use singlequote                                           quotes
  1:746   error  Requires a space before '}'                                            block-spacing
  1:747   error  Expected space(s) before "var"                                         keyword-spacing
  1:747   error  Split initialized 'var' declarations into multiple statements          one-var
  1:752   error  A space is required after ','                                          comma-spacing
  1:754   error  Operator '=' must be spaced                                            space-infix-ops
  1:757   error  A space is required after ','                                          comma-spacing
  1:759   error  Operator '=' must be spaced                                            space-infix-ops
  1:762   error  Missing whitespace after semicolon                                     semi-spacing
  1:763   error  Expected space(s) after "return"                                       keyword-spacing
  1:769   error  A space is required after '{'                                          object-curly-spacing
  1:772   error  Missing space before value for key 's'                                 key-spacing
  1:772   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:780   error  Missing space before function parentheses                              space-before-function-paren
  1:782   error  Requires a space after '{'                                             block-spacing
  1:782   error  Missing space before opening brace                                     space-before-blocks
  1:784   error  Operator '=' must be spaced                                            space-infix-ops
  1:805   error  Requires a space before '}'                                            block-spacing
  1:806   error  A space is required after ','                                          comma-spacing
  1:809   error  Missing space before value for key 'n'                                 key-spacing
  1:809   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:817   error  Missing space before function parentheses                              space-before-function-paren
  1:819   error  Requires a space after '{'                                             block-spacing
  1:819   error  Missing space before opening brace                                     space-before-blocks
  1:825   error  Operator '=' must be spaced                                            space-infix-ops
  1:834   error  Missing whitespace after semicolon                                     semi-spacing
  1:835   error  Return statement should not contain assignment                         no-return-assign
  1:843   error  Operator '=' must be spaced                                            space-infix-ops
  1:850   error  Unexpected use of comma operator                                       no-sequences
  1:850   error  A space is required after ','                                          comma-spacing
  1:852   error  Requires a space before '}'                                            block-spacing
  1:853   error  A space is required after ','                                          comma-spacing
  1:856   error  Missing space before value for key 'e'                                 key-spacing
  1:856   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:864   error  Missing space before function parentheses                              space-before-function-paren
  1:867   error  Requires a space after '{'                                             block-spacing
  1:867   error  Missing space before opening brace                                     space-before-blocks
  1:868   error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  1:869   error  Operator '=' must be spaced                                            space-infix-ops
  1:872   error  Unexpected use of comma operator                                       no-sequences
  1:872   error  A space is required after ','                                          comma-spacing
  1:874   error  Operator '=' must be spaced                                            space-infix-ops
  1:876   error  Requires a space before '}'                                            block-spacing
  1:877   error  A space is required after ','                                          comma-spacing
  1:880   error  Missing space before value for key 'f'                                 key-spacing
  1:880   error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:888   error  Missing space before function parentheses                              space-before-function-paren
  1:890   error  Requires a space after '{'                                             block-spacing
  1:890   error  Missing space before opening brace                                     space-before-blocks
  1:891   error  Expected space(s) after "try"                                          keyword-spacing
  1:894   error  Requires a space after '{'                                             block-spacing
  1:896   error  Operator '||' must be spaced                                           space-infix-ops
  1:898   error  Expected literal to be on the right side of ==                         yoda
  1:902   error  Operator '==' must be spaced                                           space-infix-ops
  1:912   error  Operator '||' must be spaced                                           space-infix-ops
  1:924   error  Requires a space before '}'                                            block-spacing
  1:925   error  Expected space(s) before "finally"                                     keyword-spacing
  1:925   error  Expected space(s) after "finally"                                      keyword-spacing
  1:932   error  Requires a space after '{'                                             block-spacing
  1:933   error  Expected space(s) after "if"                                           keyword-spacing
  1:938   error  Expected space(s) before "throw"                                       keyword-spacing
  1:938   error  Unsafe usage of ThrowStatement                                         no-unsafe-finally
  1:945   error  Requires a space before '}'                                            block-spacing
  1:946   error  Requires a space before '}'                                            block-spacing
  1:947   error  A space is required before '}'                                         object-curly-spacing
  1:948   error  Requires a space before '}'                                            block-spacing
  1:949   error  Expected a function expression                                         func-style
  1:949   error  Expected space(s) before "function"                                    keyword-spacing
  1:949   error  Use const or class constructors instead of named functions             prefer-arrow/prefer-arrow-functions
  1:959   error  Missing space before function parentheses                              space-before-function-paren
  1:961   error  A space is required after ','                                          comma-spacing
  1:964   error  Requires a space after '{'                                             block-spacing
  1:964   error  Missing space before opening brace                                     space-before-blocks
  1:966   error  Expected literal to be on the right side of ==                         yoda
  1:970   error  Operator '==' must be spaced                                           space-infix-ops
  1:973   error  Operator '||' must be spaced                                           space-infix-ops
  1:976   error  Operator '>' must be spaced                                            space-infix-ops
  1:986   error  Operator '&&' must be spaced                                           space-infix-ops
  1:990   error  Operator '=' must be spaced                                            space-infix-ops
  1:1000  error  Missing whitespace after semicolon                                     semi-spacing
  1:1001  error  Expected space(s) after "for"                                          keyword-spacing
  1:1010  error  Operator '=' must be spaced                                            space-infix-ops
  1:1012  error  A space is required after ','                                          comma-spacing
  1:1014  error  Operator '=' must be spaced                                            space-infix-ops
  1:1027  error  Missing whitespace after semicolon                                     semi-spacing
  1:1029  error  Operator '<' must be spaced                                            space-infix-ops
  1:1031  error  Missing whitespace after semicolon                                     semi-spacing
  1:1040  error  Operator '=' must be spaced                                            space-infix-ops
  1:1045  error  Missing whitespace after semicolon                                     semi-spacing
  1:1054  error  Requires a space before '}'                                            block-spacing
  1:1055  error  Expected space(s) before "var"                                         keyword-spacing
  1:1055  error  Split initialized 'var' declarations into multiple statements          one-var
  1:1060  error  Operator '=' must be spaced                                            space-infix-ops
  1:1061  error  A space is required after '{'                                          object-curly-spacing
  1:1067  error  Missing space before value for key 'dark'                              key-spacing
  1:1071  error  A space is required after ','                                          comma-spacing
  1:1078  error  Missing space before value for key 'light'                             key-spacing
  1:1082  error  A space is required after ','                                          comma-spacing
  1:1088  error  Missing space before value for key 'auto'                              key-spacing
  1:1092  error  A space is required before '}'                                         object-curly-spacing
  1:1093  error  A space is required after ','                                          comma-spacing
  1:1095  error  Operator '=' must be spaced                                            space-infix-ops
  1:1120  error  Strings must use singlequote                                           quotes
  1:1133  error  A space is required after ','                                          comma-spacing
  1:1135  error  Operator '=' must be spaced                                            space-infix-ops
  1:1160  error  Strings must use singlequote                                           quotes
  1:1176  error  A space is required after ','                                          comma-spacing
  1:1178  error  Operator '=' must be spaced                                            space-infix-ops
  1:1203  error  Strings must use singlequote                                           quotes
  1:1227  error  A space is required after ','                                          comma-spacing
  1:1229  error  Operator '=' must be spaced                                            space-infix-ops
  1:1254  error  Strings must use singlequote                                           quotes
  1:1269  error  A space is required after ','                                          comma-spacing
  1:1271  error  Operator '=' must be spaced                                            space-infix-ops
  1:1296  error  Strings must use singlequote                                           quotes
  1:1310  error  A space is required after ','                                          comma-spacing
  1:1312  error  Operator '=' must be spaced                                            space-infix-ops
  1:1337  error  Strings must use singlequote                                           quotes
  1:1360  error  A space is required after ','                                          comma-spacing
  1:1362  error  Operator '=' must be spaced                                            space-infix-ops
  1:1403  error  Strings must use singlequote                                           quotes
  1:1436  error  A space is required after ','                                          comma-spacing
  1:1438  error  Operator '=' must be spaced                                            space-infix-ops
  1:1439  error  A space is required after '{'                                          object-curly-spacing
  1:1449  error  Missing space before value for key 'fileName'                          key-spacing
  1:1473  error  Strings must use singlequote                                           quotes
  1:1491  error  A space is required after ','                                          comma-spacing
  1:1501  error  Missing space before value for key 'fileSize'                          key-spacing
  1:1525  error  Strings must use singlequote                                           quotes
  1:1543  error  A space is required after ','                                          comma-spacing
  1:1550  error  Missing space before value for key 'theme'                             key-spacing
  1:1574  error  Strings must use singlequote                                           quotes
  1:1588  error  A space is required before '}'                                         object-curly-spacing
  1:1589  error  A space is required after ','                                          comma-spacing
  1:1591  error  Operator '=' must be spaced                                            space-infix-ops
  1:1610  error  Strings must use singlequote                                           quotes
  1:1642  error  A space is required after ','                                          comma-spacing
  1:1644  error  Operator '=' must be spaced                                            space-infix-ops
  1:1645  error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:1653  error  Missing space before function parentheses                              space-before-function-paren
  1:1656  error  Requires a space after '{'                                             block-spacing
  1:1656  error  Missing space before opening brace                                     space-before-blocks
  1:1657  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  1:1658  error  Operator '=' must be spaced                                            space-infix-ops
  1:1660  error  Operator '||' must be spaced                                           space-infix-ops
  1:1672  error  Operator '?' must be spaced                                            space-infix-ops
  1:1673  error  Strings must use singlequote                                           quotes
  1:1681  error  Strings must use singlequote                                           quotes
  1:1688  error  Unexpected use of comma operator                                       no-sequences
  1:1688  error  A space is required after ','                                          comma-spacing
  1:1694  error  Operator '=' must be spaced                                            space-infix-ops
  1:1695  error  Strings must use singlequote                                           quotes
  1:1797  error  A space is required after ','                                          comma-spacing
  1:1798  error  Strings must use singlequote                                           quotes
  1:1814  error  Requires a space before '}'                                            block-spacing
  1:1815  error  A space is required after ','                                          comma-spacing
  1:1817  error  Operator '=' must be spaced                                            space-infix-ops
  1:1818  error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:1826  error  Missing space before function parentheses                              space-before-function-paren
  1:1828  error  Requires a space after '{'                                             block-spacing
  1:1828  error  Missing space before opening brace                                     space-before-blocks
  1:1829  error  Split initialized 'var' declarations into multiple statements          one-var
  1:1834  error  Operator '=' must be spaced                                            space-infix-ops
  1:1887  error  A space is required after ','                                          comma-spacing
  1:1889  error  Operator '=' must be spaced                                            space-infix-ops
  1:1890  error  Strings must use singlequote                                           quotes
  1:1900  error  Expected literal to be on the right side of ===                        yoda
  1:1900  error  Strings must use singlequote                                           quotes
  1:1906  error  Operator '===' must be spaced                                          space-infix-ops
  1:1910  error  Operator '?' must be spaced                                            space-infix-ops
  1:1911  error  Strings must use singlequote                                           quotes
  1:1920  error  A space is required after ','                                          comma-spacing
  1:1921  error  Strings must use singlequote                                           quotes
  1:1932  error  A space is required after ','                                          comma-spacing
  1:1934  error  Operator '=' must be spaced                                            space-infix-ops
  1:1935  error  Strings must use singlequote                                           quotes
  1:1945  error  Strings must use singlequote                                           quotes
  1:1970  error  Missing whitespace after semicolon                                     semi-spacing
  1:1977  error  Operator '=' must be spaced                                            space-infix-ops
  1:1979  error  Missing whitespace after semicolon                                     semi-spacing
  1:1980  error  Split initialized 'var' declarations into multiple statements          one-var
  1:1985  error  A space is required after ','                                          comma-spacing
  1:1987  error  Operator '=' must be spaced                                            space-infix-ops
  1:1992  error  Missing whitespace after semicolon                                     semi-spacing
  1:1993  error  Expected space(s) after "try"                                          keyword-spacing
  1:1996  error  Requires a space after '{'                                             block-spacing
  1:1997  error  Expected space(s) after "for"                                          keyword-spacing
  1:2006  error  Missing whitespace after semicolon                                     semi-spacing
  1:2010  error  Operator '=' must be spaced                                            space-infix-ops
  1:2024  error  Requires a space after '{'                                             block-spacing
  1:2024  error  Missing space before opening brace                                     space-before-blocks
  1:2030  error  Operator '=' must be spaced                                            space-infix-ops
  1:2038  error  Missing whitespace after semicolon                                     semi-spacing
  1:2047  error  Operator '=' must be spaced                                            space-infix-ops
  1:2048  error  Expected literal to be on the right side of ===                        yoda
  1:2050  error  Operator '===' must be spaced                                          space-infix-ops
  1:2068  error  Requires a space before '}'                                            block-spacing
  1:2069  error  Requires a space before '}'                                            block-spacing
  1:2070  error  Expected space(s) before "catch"                                       keyword-spacing
  1:2070  error  Expected space(s) after "catch"                                        keyword-spacing
  1:2078  error  Requires a space after '{'                                             block-spacing
  1:2078  error  Missing space before opening brace                                     space-before-blocks
  1:2085  error  Requires a space before '}'                                            block-spacing
  1:2086  error  Expected space(s) before "finally"                                     keyword-spacing
  1:2086  error  Expected space(s) after "finally"                                      keyword-spacing
  1:2093  error  Requires a space after '{'                                             block-spacing
  1:2099  error  Requires a space before '}'                                            block-spacing
  1:2100  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  1:2120  error  Operator '=' must be spaced                                            space-infix-ops
  1:2122  error  Unexpected use of comma operator                                       no-sequences
  1:2122  error  A space is required after ','                                          comma-spacing
  1:2143  error  Operator '=' must be spaced                                            space-infix-ops
  1:2144  error  Strings must use singlequote                                           quotes
  1:2169  error  A space is required after ','                                          comma-spacing
  1:2170  error  Strings must use singlequote                                           quotes
  1:2176  error  A space is required after ','                                          comma-spacing
  1:2177  error  Expected literal to be on the right side of ===                        yoda
  1:2177  error  Strings must use singlequote                                           quotes
  1:2183  error  Operator '===' must be spaced                                          space-infix-ops
  1:2187  error  Operator '?' must be spaced                                            space-infix-ops
  1:2192  error  A space is required after ','                                          comma-spacing
  1:2210  error  Operator '=' must be spaced                                            space-infix-ops
  1:2532  error  A space is required after ','                                          comma-spacing
  1:2550  error  Operator '=' must be spaced                                            space-infix-ops
  1:2551  error  Strings must use singlequote                                           quotes
  1:2581  error  A space is required after ','                                          comma-spacing
  1:2582  error  Strings must use singlequote                                           quotes
  1:2587  error  Requires a space before '}'                                            block-spacing
  1:2588  error  Missing whitespace after semicolon                                     semi-spacing
  1:2589  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
  1:2608  error  Strings must use singlequote                                           quotes
  1:2616  error  A space is required after ','                                          comma-spacing
  1:2619  error  Unexpected use of comma operator                                       no-sequences
  1:2619  error  A space is required after ','                                          comma-spacing
  1:2623  error  A space is required after ','                                          comma-spacing
  1:2638  error  Unnecessary parentheses around expression                              no-extra-parens
  1:2639  error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:2647  error  Missing space before function parentheses                              space-before-function-paren
  1:2649  error  Requires a space after '{'                                             block-spacing
  1:2649  error  Missing space before opening brace                                     space-before-blocks
  1:2650  error  Expected literal to be on the right side of ===                        yoda
  1:2650  error  Strings must use singlequote                                           quotes
  1:2656  error  Operator '===' must be spaced                                          space-infix-ops
  1:2672  error  Operator '&&' must be spaced                                           space-infix-ops
  1:2677  error  Requires a space before '}'                                            block-spacing
  1:2680  error  A space is required after ','                                          comma-spacing
  1:2684  error  A space is required after ','                                          comma-spacing
  1:2715  error  A space is required after ','                                          comma-spacing
  1:2735  error  Strings must use singlequote                                           quotes
  1:2742  error  A space is required after ','                                          comma-spacing
  1:2743  error  Unnecessary parentheses around expression                              no-extra-parens
  1:2744  error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:2752  error  Missing space before function parentheses                              space-before-function-paren
  1:2754  error  Requires a space after '{'                                             block-spacing
  1:2754  error  Missing space before opening brace                                     space-before-blocks
  1:2755  error  Split initialized 'var' declarations into multiple statements          one-var
  1:2760  error  Operator '=' must be spaced                                            space-infix-ops
  1:2780  error  Operator '||' must be spaced                                           space-infix-ops
  1:2798  error  A space is required after ','                                          comma-spacing
  1:2800  error  Operator '=' must be spaced                                            space-infix-ops
  1:2853  error  A space is required after ','                                          comma-spacing
  1:2855  error  Operator '=' must be spaced                                            space-infix-ops
  1:2879  error  Strings must use singlequote                                           quotes
  1:2904  error  A space is required after ','                                          comma-spacing
  1:2905  error  Strings must use singlequote                                           quotes
  1:2926  error  Missing whitespace after semicolon                                     semi-spacing
  1:2947  error  Unnecessary parentheses around expression                              no-extra-parens
  1:2948  error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:2956  error  Missing space before function parentheses                              space-before-function-paren
  1:2958  error  Requires a space after '{'                                             block-spacing
  1:2958  error  Missing space before opening brace                                     space-before-blocks
  1:2972  error  Operator '=' must be spaced                                            space-infix-ops
  1:2973  error  Strings must use singlequote                                           quotes
  1:2976  error  Requires a space before '}'                                            block-spacing
  1:2986  error  Unnecessary parentheses around expression                              no-extra-parens
  1:2987  error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:2995  error  Missing space before function parentheses                              space-before-function-paren
  1:2997  error  Requires a space after '{'                                             block-spacing
  1:2997  error  Missing space before opening brace                                     space-before-blocks
  1:3011  error  Operator '=' must be spaced                                            space-infix-ops
  1:3012  error  Strings must use singlequote                                           quotes
  1:3015  error  Requires a space before '}'                                            block-spacing
  1:3024  error  Unnecessary parentheses around expression                              no-extra-parens
  1:3025  error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:3033  error  Missing space before function parentheses                              space-before-function-paren
  1:3035  error  Requires a space after '{'                                             block-spacing
  1:3035  error  Missing space before opening brace                                     space-before-blocks
  1:3054  error  Unnecessary parentheses around expression                              no-extra-parens
  1:3055  error  Prefer using arrow functions over plain functions                      prefer-arrow/prefer-arrow-functions
  1:3063  error  Missing space before function parentheses                              space-before-function-paren
  1:3065  error  Requires a space after '{'                                             block-spacing
  1:3065  error  Missing space before opening brace                                     space-before-blocks
  1:3079  error  Operator '=' must be spaced                                            space-infix-ops
  1:3080  error  Strings must use singlequote                                           quotes
  1:3082  error  Requires a space before '}'                                            block-spacing
  1:3084  error  A space is required after ','                                          comma-spacing
  1:3089  error  Requires a space before '}'                                            block-spacing
  1:3092  error  Requires a space before '}'                                            block-spacing
  1:3095  error  Extra semicolon                                                        semi
  2:1     error  Expected space or tab after '//' in comment                            spaced-comment

✖ 446 problems (446 errors, 0 warnings)
  409 errors and 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
PS D:\Development\water.css>
```
</details>

I'm not seeing any changes when I'm looking at git, so I don't think anything in vscode broke this...

Reply:
Ooh, it's linting the compiled files in `dist/`!
We'll need to exclude this folder from all linting.